manual-node-upgrade/manual_node_upgrade.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+# Copyright 2022 Google LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     https://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Example: `./manual_node_upgrade.sh <cluster> <region>`
+#
+
+CLUSTER_NAME=$1
+REGION=$2
```

Comment:
would be a good idea to initialize with the default if not specified

Reply:
One configured by `gcloud config set compute/zone us-central1-c`

Reply:
I went with your first recommendation to halt the script execution and post an example well-formed command if either of the cluster or region variables aren't set.